pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -441,9 +441,9 @@ func (k *Kubernetes) UpdateKubernetesObjects(name string, service kobject.Servic
 
 		// Configure the container restart policy.
 		switch service.Restart {
-		case "", "always":
+		case "", "always", "any":
 			template.Spec.RestartPolicy = api.RestartPolicyAlways
-		case "no":
+		case "no", "none":
 			template.Spec.RestartPolicy = api.RestartPolicyNever
 		case "on-failure":
```

Comment:
I know, but on_failure is still supported. I can't remember where in `docker/cli` is it, but it's there. It automatically converts. Unfortunately, some examples use `on_failure`.